.github/workflows/nix.yml

```diff
@@ -1,8 +1,6 @@
 name: Nix
 
-on:
-  - pull_request
-  - push
+on: push
```

Comment:
No longer being run on a `pull_request` @rushmorem?

Reply:
Yes, I decided to run it just on `main` so that it doesn't affect the total run time. Seems like it's only me and perhaps @kearfy who use Nix at all :rofl: